spirit/comment/poll/managers.py

```diff
@@ -3,7 +3,7 @@
 from __future__ import unicode_literals
 
 from django.db import models
-from django.db.models.loading import get_model
+from django.apps.config import detached AppConfig
```

Comment:
This should be `from django.apps import apps` then `apps.get_model(...)`

btw, what is `detached`?? seems like some obscure Python 3 feature, it raises a syntax error in python 2. I'm very curious about this.